pytorch_lightning/trainer/training_loop.py

```diff
@@ -491,16 +491,14 @@ def optimizer_step(self, optimizer, opt_idx, batch_idx, train_step_and_backward_
 
         # model hook
         model_ref.optimizer_step(
-            epoch=self.trainer.current_epoch,
-            batch_idx=batch_idx,
-            optimizer=optimizer,
-            optimizer_idx=opt_idx,
-            optimizer_closure=train_step_and_backward_closure,
+            self.trainer.current_epoch,
```

Comment:
why do this? If it's explicit it reduces the risk of potential errors in ordering imo

Reply:
Because It was breaking backward compatibility.

Reply:
thanks!